src/domains/transaction/components/AddRecipient/AddRecipient.tsx

```diff
@@ -57,24 +57,14 @@ const TransferType = ({ isSingle, disableMultiple, onChange, maxRecipients }: To
 	);
 };
 
-const InputButtonStyled = styled.button(() => [
-	tw`flex items-center h-full px-5 font-semibold text-theme-secondary-700`,
-	tw`border-2 rounded border-theme-primary-100`,
-	tw`transition-colors duration-300`,
-	tw`dark:(border-theme-secondary-800 text-theme-secondary-500)`,
-	tw`focus:(outline-none ring-2 ring-theme-primary-400)`,
-	tw`hover:(border-theme-primary-100 bg-theme-primary-100 text-theme-primary-700)`,
-	tw`dark:hover:(border-theme-secondary-800 bg-theme-secondary-800 text-white)`,
-	tw`disabled:(
-		border border-theme-secondary-300 text-theme-secondary-500 cursor-not-allowed
-		dark:(border-theme-secondary-700 text-theme-secondary-700)
-	)`,
-	css`
-		&.active {
-			${tw`border-theme-primary-600 bg-theme-primary-50 dark:bg-theme-primary-900 dark:text-white`}
-		}
-	`,
-]);
+const InputButtonStyled = ({ ...props }: React.ButtonHTMLAttributes<HTMLButtonElement>) => (
+	<button
+		{...props}
+		className={twMerge(
```

Comment:
I appreciate the work you are doing, however, I'd be cautious with the `twMerge` because https://github.com/dcastil/tailwind-merge/blob/v2.5.5/docs/when-and-how-to-use-it.md#reasons-not-to-use-it

I know there are some cases that we have to use `twMerge` but still I'd use `cn` whenever possible. Please take a look at the link I shared above.